ci/check-rust-version.sh

```diff
@@ -1,49 +1,22 @@
-#!/bin/bash
+#!/bin/sh
 
-min_rust=(1 53 0)
-rust_ver=()
-
-parse_rustc_version() {
-  local IFS
-  IFS=' '
-  local FIELDS
-  read -ra FIELDS <<< $(rustc --version)
-  IFS='.'
-  read -ra rust_ver <<< "${FIELDS[1]}"
-}
+min_rust="1.53.0"
+rust_ver="$(rustc --version | cut -d' ' -f2)"
 
 check_rust_version() {
-  parse_rustc_version
-  # rust_ver=(1 46 0) for testing purposes
-
-  if [[ "${rust_ver[0]}" -gt "${min_rust[0]}" ]] ; then
+  ver=$(printf "%s\n%s\n" "$min_rust" "$rust_ver" | sort -V | head -n1)
```

Comment:
ah, pretty neat!
I generally prefer the longer names for relatively unusual parameters; assuming that the long name is portable enough, can we use this?

```suggestion
  ver=$(printf "%s\n%s\n" "$min_rust" "$rust_ver" | sort --version-sort | head -n1)
```

Reply:
Nope, longer names are a GNU thing, and most posix environments will have only `-V` (e.g. busybox)

Reply:
Probably most if not all development machines will have gnu coreutils. On Alpine you can just install it via ``apk add coreutils`` which replaces the limited busybox sort applet. It will be so much easier for everyone if we would use that and wouldn't bring other people in the pity to break Alpine compatibility by accident.
I think it would be a really good choice to do the following for alpine in beginning of ``get-deps``:
```
sudo apk add coreutils bash
```

Reply:
`sudo` is deprecated and doesn't have to be installed everywhere (see line 9-15), `apk` is alpine specific, you can have universal script across systems at very minimal cost

> Probably most if not all development machines will have gnu coreutils.

The script doesn't have to be used on development machine. If someone just wants to use/install `wezterm`, they don't need `bash` nor `coreutils`.

Reply:
While I'm still in the process of fixing `wezterm` builds on Alpine (ref: https://gitlab.alpinelinux.org/panekj/aports/-/pipelines/105982), reasons you can't use provided binaries are:
- AppImage doesn't work with musl
- No pre-built binaries are provided for musl
- `wezterm` can't be built statically on musl due to proc-macros, thus it's not portable across musl systems